src/components/LanguageSwitcher.module.css

```diff
@@ -0,0 +1,11 @@
+/* TODO: BEMS convention */
```

Comment:
- I think we should avoid using css modules and instead use our themes and classNames 
- Each time we merge to dev, I think it would be best to already be using the BEM naming conventions. That way we don't have to change the classNames in PRs that are unrelated or worry about forgetting to change them

Reply:
These CSS modules are for when we need to use custom styling that is not covered in USWDS. Did you want to use a simple stylesheet instead? Also, I was trying to use the BEMS convention with CSS modules and it would not like the hyphens in class selectors when incorporating in HTML, so I figured that I would leave that comment to come back to it later.

Reply:
I think we'd want to add classes in a scss file for custom styling, not css

Reply:
It should also probably be in our styles folder (which I created in the branch I'm working on today), instead of components

Reply:
I believe Justin said a while back to use CSS files for custom classes.